src/components/VoteBox.jsx

```diff
@@ -0,0 +1,45 @@
+import upvote from "../assets/NC-UPVOTE.jpg";
+import downvote from "../assets/NC-DOWNVOTE.jpg";
+import { useState } from "react";
+import { patchVotes } from "../utils/api";
+
+export default function VoteBox({ votes, article_id }) {
+  const [userVote, setUserVote] = useState(0);
+  const [err, setErr] = useState(null);
+
+  const handleClick = (vote) => {
+    setUserVote((currentVotes) => currentVotes + vote);
+    setErr(null);
+    patchVotes(article_id, vote).catch((err) => {
+      setUserVote((currentVotes) => currentVotes - vote);
+      setErr("Something went wrong, please try again");
+    });
+  };
+
+  return (
+    <>
+      <section className="vote-box">
+        <button
+          className={`vote-button upvote `}
+          onClick={() => {
+            handleClick(1);
+          }}
+          disabled={userVote > 0}
+        >
+          <img src={upvote} alt="Upvote arrow using nc logo"></img>
```

Comment:
Good accessibility on the img alt tags! I would suggest having a look into aria-labels on the button to tell a screen reader the action that each of these buttons take :D